webapp/tests/client_websocket.test.jsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+/*
```

Comment:
Mentioned in my PR summary, but the unit tests don't have an implementation of WebSocket and I was running into issues including one. Going to look into it later when I have more time